Modules/Loadable/Markups/MRML/vtkMRMLMarkupsAngleNode.cxx

```diff
@@ -33,6 +33,8 @@ vtkMRMLNodeNewMacro(vtkMRMLMarkupsAngleNode);
 
 //----------------------------------------------------------------------------
 vtkMRMLMarkupsAngleNode::vtkMRMLMarkupsAngleNode()
+  : AngleMode(0)
```

Comment:
I am also wondering if `AngleMode` should be changed to `AngleMeasurementMode`

Reply:
AngleMode is shorter than AngleMeasurementMode, and it is not strictly just measurement but display, too (angle is displayed slightly differently depending on the mode). However, if you don't find it obvious what "AngleMode" refers to then I agree that a longer, more specific name could be better.

@pieper would "AngleMode" self-explaining enough for you or do you think "AngleMeasurementMode" would be more clear?